data/blog/enjoyable-database-testing.md

```diff
@@ -0,0 +1,277 @@
+You've written some Haskell code and it compiles ... so it works?
+
+Hahahahahahahaha ha ... sigh ... *[muffled sobbing]*.
+
+Turns out this is not the case. Not the case in general and definitely not the case if you are writing database queries.
```

Comment:
urns out this is not the case - not the case in general and definitely not the case if you are writing database queries.